static/js/components/ProfileImage.js

```diff
@@ -7,18 +7,39 @@ import type { Profile } from '../flow/profileTypes';
 
 export default class ProfileImage extends React.Component {
   props: {
-    profile: Profile
+    profile: Profile,
+    editable: boolean
   };
 
   render () {
-    const { profile } = this.props;
+    const { profile, editable } = this.props;
     const imageUrl = makeProfileImageUrl(profile);
-    return (
+
+    let img = (
       <img
         src={imageUrl}
         alt={`Profile image for ${getPreferredName(profile, false)}`}
         className="card-image"
       />
     );
+    if (editable) {
+      img = (
+        <div className="avatar">
+          <img
+            src={imageUrl}
+            alt={`Profile image for ${getPreferredName(profile, false)}`}
+            className="card-image"
+          />
+          <span className="img">
+            <img
+              src='/static/images/camera.png'
+              alt="Camera image for"
+            />
+          </span>
+        </div>
+      );
+    }
+
+    return img;
```

Comment:
What do you think about refactoring this a little? Instead of declaring the `img` item twice you could write a method that checks for `editable` and, if it's set to `true`, returns the camera icon (which, again, should be an MDL `Icon`). Then you can just call that method inline.